### DISCUSSION TITLE
Disable auto image scroll in gallery view [Jun 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5975] [Discord Threads]
> 🙋‍♂️ **jewelley**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Right now, there is no such provision AFAIK. Will check with team and confirm  
#  
> 🙋‍♂️ **jewelley**  
#### Thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1109191479135584367)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by Joonas12334** [comment]
Any updates? We really like drag and drop from baserow browser window into another upload box. It would be really helpful to keep using the image viewer this way.
